src/pcapi/scripts/interact.py

```diff
@@ -50,8 +50,16 @@ def set_python_prompt():
         color = "\x1b[1;49;36m"  # cyan
     else:
         color = None
+
     if color:
-        sys.ps1 = f"{color}{env} >>>\x1b[0m "
+        COLOR = _non_printable(color)
+        RESET = _non_printable("\x1b[0m")
```

Comment:
```suggestion
        color = _non_printable(color)
        reset = _non_printable("\x1b[0m")
```

autant réutiliser les variables (et ne pas définir des variables en majuscules)